### PR TITLE
Feature/Nav

### PR DESCRIPTION
**Pull request by chrisaddams**
Adds Navigation, includes bootstrap, makes use of liquid component includes, Adds About Us page.

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -1,44 +1,24 @@
...
-			</div>
-		</main>
-	</body>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include head.html %}
+  </head>
+  <body style="padding-top: 0; margin: 0">
+    {% include header.html %}
```
Comment (fakesamgregory) [inline 404.html:10]: This conflicts with Pinegrow. We either use liquid or Pinegrow components (which it already was a Pinegrow component)

File: UserLib.js [hunk 2]
```diff
@@ -1,23 +0,0 @@
-$(function() {
```
Comment (fakesamgregory) [inline UserLib.js:1]: Why have you deleted this? We need this.

File: _data/whatis.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+content: <p>The podcast that takes a deeper look into all things tech with
+  those who make it happen. Weekly we talk about the latest
+  developments in technology around the world, and we have a
+  fantastic guest talking about the part they play in the tech
+  industry and how it's shaped their lives.</p>
+  <br/>
```
Comment (fakesamgregory) [inline _data/whatis.yml:6]: Not a correct use of `br` We should use CSS on p tags to make this a universal style for paragraphs following paragraphs

**Review by fakesamgregory — COMMENTED** [review]
We need to settle on what we use. Pinegrow has a concept of components that we were already using. Using Jekyll components doesn't work within Pinegrow...

File: UserLib.js (hunk 2 above)
Comment (chrisaddams) [reply UserLib.js:1]: I thought I'd created that by mistake in pinegrow😂
I can ad that back in

File: _data/whatis.yml (hunk 3 above)
Comment (chrisaddams) [reply _data/whatis.yml:6]: fair enough, can change that. I haven't really written HTML in a while...

**Comment by fakesamgregory** [comment]
@chrisaddams Other than some questionable CSS this looks fine. Nothing looks broken

**Pull request merged**